daprdocs/content/en/operations/components/setup-secret-store/supported-secret-stores/azure-keyvault.md

```diff
@@ -40,13 +40,25 @@ The above example uses secrets as plain strings. It is recommended to use a loca
 
 ## Spec metadata fields
 
+### Self-Hosted
+
 | Field              | Required | Details                                                                 | Example                  |
 |--------------------|:--------:|-------------------------------------------------------------------------|--------------------------|
 | vaultName          | Y        | The name of the Azure Key Vault                                         | `"mykeyvault"`           |
 | spnTenantId        | Y        | Service Principal Tenant Id                                        | `"spnTenantId"`          |
 | spnClientId        | Y        | Service Principal App Id                                           | `"spnAppId"`             |
 | spnCertificateFile | Y        | PFX certificate file path. <br></br> For Windows the `[pfx_certificate_file_fully_qualified_local_path]` value must use escaped backslashes, i.e. double backslashes. For example `"C:\\folder1\\folder2\\certfile.pfx"`. <br></br> For Linux you can use single slashes. For example `"/folder1/folder2/certfile.pfx"`.  <br></br> See [configure the component](#configure-the-component) for more details | `"C:\\folder1\\folder2\\certfile.pfx"`, `"/folder1/folder2/certfile.pfx"`        |
 
+
+### Kubernetes
+
+| Field              | Required | Details                                                                 | Example                  |
+|--------------------|:--------:|-------------------------------------------------------------------------|--------------------------|
+| vaultName          | Y        | The name of the Azure Key Vault                                         | `"mykeyvault"`           |
+| spnTenantId        | Y        | Service Principal Tenant Id                                        | `"spnTenantId"`          |
+| spnClientId        | Y        | Service Principal App Id                                           | `"spnAppId"`             |
+| spnCertificate     | Y        | Store the certificate for the service principal into the Kubernetes Secret Store.<br></br> See [configure the component](#configure-the-component) for more details | kubectl create secret generic [name] --from-file=[fully_qualified_local_path]        |
```

Comment:
Also did we make a change to the component so that `spnCertificate` is a valid field? If so `spnCertificateFile` wouldn't be required.

Reply:
Both spnCertificate and spnCertificateFile have always been there. But unless you clicked on the Kubernetes tab at the bottom of the page there was no way to know that spnCertificate was even an option. I was trying to surface the existence of the field sooner in the doc. When I was working with this component last week I was stuck until @msfussell pointed to that tab. I wasted a lot of time not knowing there was another option to get the data to the component.